mmv1/products/spanner/Database.yaml

```diff
@@ -153,10 +153,16 @@ properties:
       - !ruby/object:Api::Type::String
         name: 'kmsKeyName'
         immutable: true
-        required: true
```

Comment:
Same, should be https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#exactly_one_of